### ISSUE TITLE
Screen reader doesn't read submenus in the primary navigation

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1966]
It says there are submenus, but I haven't figured out how to access them and they aren't in the list of elements. 
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/0f0e3d70-3b81-4744-95f7-8d6f20e73e03)

A work around for this could be to make the primary nav items clickable and bring up the section homepage.
